docs/reference/specifications/in-process-providers.md

```diff
@@ -94,6 +95,15 @@ This evaluator selects a variant based on whether the specified property within
 starts/ends with a certain string.
 For more specific implementation guidelines, please refer to [this document](./custom-operations/string-comparison-operation-spec.md).
 
+### $flagd properties in the evaluation context
+
+An in-process flagd provider should also add the following properties to the JsonLogic evaluation context so that users can use them in their targeting rules.
```

Comment:
Could you briefly describe how these values are merged with the evaluation context?

Reply:
I've added a suggestion: https://github.com/open-feature/flagd/pull/965/files#r1357396324

Reply:
Updated as per Todd's suggestion. Thanks again!